components/GetstartedCard2/data.ts

```diff
@@ -0,0 +1,19 @@
+export interface CardWithImage {
```

Comment:
let's take out this data.ts file. The content here is too little to create a file for. Please put the content on the jsx directly.